src/prisma/schema.prisma

```diff
@@ -2972,6 +2972,7 @@ model UserWeeklyStats {
   user          Scout    @relation(fields: [userId], references: [id], onDelete: Cascade)
   gemsCollected Int
   week          String
+  rank          Int      @default(0)
```

Comment:
is the plan to make this required with no default in a second migration? i was saying that i think you can do it in one PR - just now you need to remove the default, and generate a second prisma migration